pkg/resourceinterpreter/interpreter.go

```diff
@@ -111,6 +115,13 @@ func (i *customResourceInterpreterImpl) GetReplicas(object *unstructured.Unstruc
 	if hookEnabled {
 		return
 	}
+	replica, requires, hookEnabled, err = i.thirdpartyInterpreter.GetReplicas(object)
+	if err != nil {
+		return
+	}
+	if hookEnabled {
+		return
+	}
```

Comment:
Should we put the `thirdpartyInterpreter` after the `defaultInterpreter` for better efficiency?
I think K8s native resources are used more often than third-party ones.

Reply:
Same as below.

Reply:
The resources interpreted by `thirdpartyInterpreter` and `thirdpartyInterpreter` maybe not overlap, and I agree with this point.

Reply:
> The resources interpreted by thirdpartyInterpreter and thirdpartyInterpreter maybe not overlap

Yes, that's exactly what I was thinking. 
@chaunceyjiang What do you think?

Reply:
In the `ReflectStatus` of `defaultInterpreter`, if there is no built-in handler, it will attempt to collect the whole status from '.status' filed.

Refter to: https://github.com/karmada-io/karmada/blob/master/pkg/resourceinterpreter/default/native/default.go#L134-L135

If `thirdpartyInterpreter` is placed after `defaultInterpreter`, then the `ReflectStatus` of `thirdpartyInterpreter` will not take effect.

Reply:
@RainbowMango What do you think?

Reply:
> If thirdpartyInterpreter is placed after defaultInterpreter, then the ReflectStatus of thirdpartyInterpreter will not take effect.

Good point!

Reply:
I missed that, thank you.

/approve